dist/configs/rust-in-replit.yml

```diff
@@ -0,0 +1,15 @@
+"project_id_env": "CROWDIN_PROJECT_ID"
+"api_token_env": "CROWDIN_PERSONAL_TOKEN"
+"base_path_env": "CROWDIN_BASE_PATH"
+"base_url_env": "CROWDIN_BASE_URL"
+
+"preserve_hierarchy": true
+
+files:
+  [
+    {
+      "source": "/tooling/locales/en/*.md",
```

Comment:
I think I need to remove this, because I also want the `comments.json` file to be translated. @nhcarrigan I would like to meet to get a run-down for how this magic works.

```suggestion
      "source": "/tooling/locales/en/*",
```